### PR TITLE
Bump version to v3.2.0

### PR DESCRIPTION
**Pull request by rsanchez15**

### CODE REVIEW

**Review by Mario-DL — APPROVED** [review]
LGTM

**Review by Mario-DL — CHANGES_REQUESTED** [review]
I think we should update this https://github.com/eProsima/vulcanexus/blob/25b3b5b6562d09cdf94e9f3b58b4ef61729756c6/docs/rst/notes/notes.rst#L29
and the previous notes versions

File: docs/rst/notes/iron/notes.rst [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .. _notes_iron_latest:
```
Comment (Mario-DL) [inline docs/rst/notes/iron/notes.rst:1]: I am missing here the `https://github.com/eProsima/rosidl.git` new package

**Review by Mario-DL — CHANGES_REQUESTED** [review]
As we were commenting internally, humble versions should also be updated

File: docs/rst/notes/iron/notes.rst (hunk 1 above)
Comment (rsanchez15) [reply docs/rst/notes/iron/notes.rst:1]: Added in 6d3682d

**Review by Mario-DL — APPROVED** [review]
LGTM

**Pull request merged**